docs/deploying-kas-fleet-manager-to-openshift.md

```diff
@@ -110,6 +111,9 @@ make deploy/secrets <OPTIONAL_PARAMETERS>
 - `OBSERVABILITY_RHSSO_METRICS_CLIENT_ID`: The client id for a RHSSO service account that has read metrics permission. Defaults to `''`
 - `OBSERVABILITY_RHSSO_METRICS_SECRET`: The client secret for a RHSSO service account that has read metrics permission. Defaults to `''`
 - `JWKS_VERIFY_INSECURE`: Skip TLS insecure verification for the connection for fetching jwks certificate. Defaults to value false.
+- `ACME_ISSUER_ACCOUNT_KEY`: The ACME Issuer account key used for the automatic management of certificate. This is required when certificate management mode is `automatic`. Defaults to `''`
+- `AWS_SECRET_MANAGER_SECRET_ACCESS_KEY`: AWS secret manager secret access key: Defaults to `''`. This is required when certificate management mode is `automatic`.
+- `AWS_SECRET_MANAGER_ACCESS_KEY`: AWS secret manager access key: Defaults to `''`. This is required when certificate management mode is `automatic`.
```

Comment:
How do you control the certificate management mode? As a user, reading this line I end up with that question

Reply:
same applies to other flags/params

Reply:
I think related configurations are documented right below e.g KAFKA_TLS_CERTIFICATE_MANAGEMENT_STRATEGY